pandora-client-web/src/assets/appearanceValidation.ts

```diff
@@ -79,6 +79,8 @@ export function RenderAppearanceActionResult(assetManager: AssetManagerClient, r
 				return `The ${DescribeAsset(assetManager, e.asset)} cannot be added, removed, or modified, because ${DescribeAssetSlot(assetManager, e.slot)} is covered by another item.`;
 			case 'blockedHands':
 				return `You need to be able to use hands to do this.`;
+			case 'exitPose':
```

Comment:
Is this considered to be a debugging fail-safe in case some asset creator specified an exit pose that is impossible to assume under certain conditions?

What happens in this case? They cannot exit the device? Can someone else exit the character or are they stuck?

Reply:
... or maybe someone adds new items to the character while inside the slot that makes one unable to exit the device? Like making on unable to stand up from a chair.

Reply:
There are currently no checks, so you can create an asset that is impossible to exit.

We could introduce an extra check later that ensures you can assume the exit pose before entering a device. This way, if the exit pose is impossible, you can not even enter.
 
 `optional:` part of the preset can always be used as a convenience, like assume a regular arm pose exiting a cross, without being a requirement.
 
 And the intended usage for the required exit pose is indeed to allow assets restricting the pose, like sitting to keep the player tied down to a chair

Reply:
Added the missing safemode check, so even if the asset is faulty, you can exit it in safemode

Reply:
Great, thanks for explaining.
Yes, this makes sense to me.
I think it is fine to not check if you can exit when enter, since things can change while inside the slot, and with your last change, wrongly defined assets can be left with safemode in the worst case, if we cannot spot such things during reviews or tests.